content/en/docs/setup/independent/setup-ha-etcd-with-kubeadm.md

```diff
@@ -1,7 +1,7 @@
 ---
 reviewers:
 - sig-cluster-lifecycle
-title: Set up a Highly Availabile etcd Cluster With kubeadm
+title: Set up a Highly Available etcd Cluster With kubeadm
```

Comment:
thanks for the typo fix. to be fair this should be:
> Set up a High Availability etcd cluster with kubeadm

as per the HA abbreviation.

/assign @Bradamant3

Reply:
+1

@mauilion could you please edit the PR, then LGTM :)

Reply:
Corrected. Thanks!